### PR TITLE
  Doc: Updating the installation guide in the user guide to include instructions for the 'install-full' and 'install-conf' options v1. #6342

### PR DESCRIPTION
**Pull request by comfort619**
- [x] I have read the contributing guide lines at https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/install.rst) to reflect the changes made 

Link to [redmine](https://redmine.openinfosecfoundation.org/issues/6342) ticket:

Describe changes:

- Added instructions for the 'install-full' option to the installation guide.
- Included guidance on using the 'install-conf' option for better configuration.
- Ensured that these instructions are presented clearly in the appropriate sections of the guide.
- Checked and maintained consistent formatting throughout the document.

**Testing Done:**

I've tested these changes locally to ensure that the installation guide is accurate and easy to follow.

**Related Issues:**

Ticket: 6342

**Additional Notes:**

- This PR includes updates to the user guide for improved clarity.
- It follows the project's coding and documentation standards.

I look forward to your feedback and appreciate your review of these changes.

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -361,6 +361,34 @@ Suricata packages.
 
     yay -S suricata
 
+
+Auto-Setup
```
Comment (inashivb) [inline doc/userguide/install.rst:365]: I think this should go before the section where distro specific installations start.

**Review by inashivb — CHANGES_REQUESTED** [review]
Hi @comfort619 !
Kudos on making your first contribution! :)
We need several changes w.r.t. the commits. Please read the contribution guideline: https://docs.suricata.io/en/latest/devguide/codebase/contributing/code-submission-process.html#commits
1. Make commit messages small
2. Keep commit history clear (here 1 commit should suffice)
3. Make sure your author is `FirstName LastName <myemail.example.com>` and it matches your name on the contribution agreement.

**Comment by comfort619** [comment]
> Hi @comfort619 ! Kudos on making your first contribution! :) We need several changes w.r.t. the commits. Please read the contribution guideline: https://docs.suricata.io/en/latest/devguide/codebase/contributing/code-submission-process.html#commits
> 
> 1. Make commit messages small
> 2. Keep commit history clear (here 1 commit should suffice)
> 3. Make sure your author is `FirstName LastName <myemail.example.com>` and it matches your name on the contribution agreement.

okay, noted. thanks

**Comment by jufajardini** [comment]
Hi there @comfort619, thanks for working on the feedback we've given.

We usually incorporate new changes into new PRs, in our contribution process: https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html#feedback

**Comment by comfort619** [comment]
> Hi there @comfort619, thanks for working on the feedback we've given.
> 
> We usually incorporate new changes into new PRs, in our contribution process: https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html#feedback
> 
> As the changes here are not code-related, I'm re-running the checks. The points indicated by Shivani seem to have been addressed 🎉 :)

ohh, I missed that of creating a new PR for changes, noted. Thank you for the feedback

**Review by jufajardini — CHANGES_REQUESTED** [review]
Comparing with the page on https://docs.suricata.io/en/latest/devguide/codebase/installation-from-git.html I've noticed that we were using italics to differentiate the commands we were explaining.
I've indicated two cases, can you double-check what is missing, and update accordingly? :)

**Pull request closed without merging**